README.md

```diff
@@ -8,3 +8,4 @@
 - [Read a file](read.go)
 - [Read a range of a file](read.go)
 - [Read a file with callback](read.go)
+- [Append an existing or new file](append.go)
```

Comment:
We need to split them into two examples:

- Append to a new file (create_append + write_append)
- Append to an existing file (stat + write_append)